opentimelineio/console/otioconvert.py

```diff
@@ -100,6 +101,16 @@ def _parsed_args():
         'key=value. Values are strings, numbers or Python literals: True, '
         'False, etc. Can be used multiple times: -a burrito="bar" -a taco=12.'
     )
+    parser.add_argument(
+        '-A',
+        '--output-adapter-arg',
+        type=str,
+        default=[],
+        action='append',
+        help='Extra arguments to be passed to output adapter in the form of '
+        'key=value. Values are strings, numbers or Python literals: True, '
+        'False, etc. Can be used multiple times: -a burrito="bar" -a taco=12.'
```

Comment:
The example should use `-A` instead of `-a`
```suggestion
        'False, etc. Can be used multiple times: -A burrito="bar" -A taco=12.'
```